knowledge-base/implementation-documentation/0005-api-improvements.md

```diff
@@ -214,6 +214,54 @@ Cons:
 - Difficult to implement.
 - Not realistic for 2.0, but maybe later.
 
+## Function style filtering [Updated]
```

Comment:
[q] Is this the alternative 5 after the 4 alternatives, as this is also related to the filter issue?
If so, should we mark this as a winner?

Reply:
yes if this is the decision we should remove the other alternatives above.

Reply:
We didn't decide on a specific solution in the meeting, so these are still alternatives. The investigation [follow-up ticket](https://github.com/SAP/cloud-sdk-backlog/issues/854) was created instead